kobo/hub/migrations/0004_auto_20200213_0538.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-02-13 05:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
The commit message mentions:

> all the downstream projects using kobo-hub would have to handle this migration process by themselves

Question: what is the impact on those projects now when this migration appears?  Does it conflict in any way with what they've had to do up to now?

Reply:
I took a closer look as to what actually happens in the new migration and I have solved it without needing to introduce a new migration file. So now it should pose no problems in the downstream migrations.